doc/news/changes/minor/20170809ArndtMaierPelteret

```diff
@@ -0,0 +1,4 @@
+Fixed: Debugged and fixed a long standing linker error due to missing
+instantiations of ShapeInfo::reinit variants.
```

Comment:
I guess we should remove the 'long standing', yes. Can you do it? (I'm away from my computer now...)